src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js

```diff
@@ -357,7 +357,7 @@
             {
                 isValueValidator: false,
                 selector: `${SELECTOR_FIELD} ${SELECTOR_ADDRESS_INPUT}`,
-                eventName: 'addressNotFound',
+                eventName: 'ez-address-not-found',
```

Comment:
Can be a `const` maybe?

Reply:
You mean, move it above class definition? :)

Reply:
Right

Reply:
Still no `const`?